user-guide/settings/privacy.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Privacy
+slug: privacy-submenu
+tags: privacy, delete account, unsubscribe, email list
+publish: true
+position: 40
+---
+
+# Privacy submenu
+
+The __Privacy__ submenu provide options to opt-out of receiving emails from Progress and its partners and the option to delete your Fiddler Everywhere associated account.
+
+There are two options in the submenu:
+
+- __Unsubscribe from emails__
+- __Delete your Fiddler Account__
+- __Submit anonymous usage statistics__ checkbox
+
+## Unsubscribe from emails
+
+To sumbit a request to unsubscribe from receiving future emails, click on the __Unsibscribe__ link. You will be redirected to a unsubscribe page where you should follow the given instructions.
+
+![Unsubscribe from emails](../../images/settings/privacy-unsub.png)
+
+## Delete your Fiddler Account
+
+To sumbit a request to click on the __Submut "Delete Account" request__ link. You will be reditected to a page where you should follow the given instructions.
+
+![Unsubscribe from emails](../../images/settings/privacy-delete.png)
+
+## Submit anonymous usage statistics
```

Comment:
On the images the setting is called "send anonymous...". Unless this will be changed, you should rephrase it to match the actual UI.